README.md

```diff
@@ -48,7 +48,7 @@ linting.on("done", () => {
 ## Config
 
 In order to specify what should be linted SVGLint must be given a configuration object.
-If you are using the CLI, this configuration object is read from the file specified by `--config`. This defaults to `.svglintrc.js`, which will be searched for up through the directory tree - this is similar to tools such as ESLint.
+If you are using the CLI, this configuration object is read from the file specified by `--config`. This defaults to `.svglintrc.js`, which will be searched for up through the directory tree - this is similar to tools such as ESLint. If no configuration file is found searching up the directory tree, a configuration file located in the user's home directory is used (i.e. "$HOME/.svglintrc.js" on Unix-like systems).
```

Comment:
- I think we should put _"this is similar to tools such as ESLint"_ at the end because the homedir behavior is also similar. This also conveniently allows for dropping the connecting text
- I think _"`"$HOME/.svglintrc.js"`_ on Unix-like systems` is an example. It should also work on Windows.
- I'd suggest using code notation for the path.
- I would personally also prefer `~` over `$HOME`, but feel free to keep `$HOME` if you want.

```suggestion
If you are using the CLI, this configuration object is read from the file specified by `--config`. This defaults to `.svglintrc.js`, which will be searched for up through the directory tree, or in the user's home directory (e.g. `~/.svglintrc.js` on Unix-like systems) - this is similar to tools such as ESLint
```

Reply:
The README should be updated now.